src/main/java/uk/gov/hmcts/reform/prl/controllers/CallbackController.java

```diff
@@ -253,15 +256,47 @@ public AboutToStartOrSubmitCallbackResponse generateDocumentSubmitApplication(
         return AboutToStartOrSubmitCallbackResponse.builder().data(caseDataUpdated).build();
     }
 
+    @PostMapping(path = "/amend-court-details/about-to-start", consumes = APPLICATION_JSON, produces = APPLICATION_JSON)
+    @Operation(description = "Callback to Issue and send to local court")
+    @SecurityRequirement(name = "Bearer Authentication")
+    public AboutToStartOrSubmitCallbackResponse amendCourtAboutToStart(
+        @RequestHeader(HttpHeaders.AUTHORIZATION) @Parameter(hidden = true) String authorisation,
+        @RequestBody uk.gov.hmcts.reform.ccd.client.model.CallbackRequest callbackRequest) throws Exception {
+
+        CaseData caseData = CaseUtils.getCaseData(callbackRequest.getCaseDetails(), objectMapper);
+
+        Map<String, Object> caseDataUpdated = callbackRequest.getCaseDetails().getData();
+        List<DynamicListElement> courtList = locationRefDataService.getCourtLocations(authorisation);
+        caseDataUpdated.put("courtList", DynamicList.builder().value(DynamicListElement.EMPTY).listItems(courtList)
+            .build());
+        return AboutToStartOrSubmitCallbackResponse.builder().data(caseDataUpdated).build();
```

Comment:
Move this to Service

Reply:
follow this - https://github.com/hmcts/prl-cos-api/pull/584#discussion_r1066201252